client/allocrunner/networking_bridge_linux.go

```diff
@@ -156,69 +165,61 @@ func (b *bridgeNetworkConfigurator) Teardown(ctx context.Context, alloc *structs
 	return b.cni.Teardown(ctx, alloc, spec)
 }
 
-func buildNomadBridgeNetConfig(b bridgeNetworkConfigurator, withConsulCNI bool) []byte {
-	var consulCNI string
-	if withConsulCNI {
-		consulCNI = consulCNIBlock
+func buildNomadBridgeNetConfig(b bridgeNetworkConfigurator, withConsulCNI bool) ([]byte, error) {
+	// Update website/content/docs/networking/cni.mdx when the bridge configuration
```

Comment:
This function is quite lengthy since it defines all the plugins and appends them to the overall network config, I was debating adding constructors for each plugin in the config file I created, but didn't want to overcomplicate things, but I am completely open to splitting this up!

Reply:
seems like a reasonable function length to me!